test/org/loklak/harvester/TwitterScraperTest.java

```diff
@@ -33,7 +31,7 @@ public void testPrepareSearchURL() {
             url = TwitterScraper.prepareSearchURL(query[i]);
 
             //compare urls with urls created
-            assertEquals(url, out_url[i]);
+            assertThat(out_url[i], is(url));
```

Comment:
Should it not be `assertThat(url, is(equalTo(out_url[i])));`, as the convention is `assertThat(actual, is(equalTo(expected)));`